specification/sql/resource-manager/Microsoft.Sql/preview/2020-08-01-preview/RestorableDroppedDatabases.json

```diff
@@ -166,20 +166,32 @@
     },
     "RestorableDroppedDatabase": {
       "description": "A restorable dropped database resource.",
-      "required": [
-        "location"
-      ],
       "type": "object",
       "allOf": [
         {
-          "$ref": "../../../common/v1/types.json#/definitions/TrackedResource"
```

Comment:
Why are we switching this from a tracked resource to a proxy resource?

Reply:
it was to fix the issue of 

❌ R3026 - TrackedResourcePatchOperation | Tracked resource 'RestorableDroppedDatabase' must have patch operation that at least supports the update of tags. It's strongly recommended that the PATCH operation supports update of all mutable properties as well.New: Microsoft.Sql/preview/2020-08-01-preview/RestorableDroppedDatabases.json#L167
-- | --

in https://msdata.visualstudio.com/Database%20Systems/_git/DsMainDev/pullrequest/508254

Reply:
this error was generated by https://github.com/Azure/azure-rest-api-specs/pull/12107/checks?check_run_id=1765298546

Reply:
Are the owners of the RestorableDroppedDatabases resource type ok with this? This change is not without consequences...

Reply:
the change is from the owner at our request. just invited you to the group chat for more context.